streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -457,8 +463,9 @@
                                        final String... stateStoreNames);
 
     /**
-     * Create a new {@code KTable} by transforming the value of each record in this {@code KTable} into a new value,
-     * (with possibly new type).
+     * Create a new {@code KTable} by transforming the value of each record in this {@code KTable} into a new value
+     * (with possibly new type), using the {@link Materialized} instance for configuration of the {@link Serde key serde},
```

Comment:
super nit: `with possibly new type` -> `with possibly a new type`

Here and a few other places.